src/app/services/cell.service.ts

```diff
@@ -15,10 +15,16 @@ export class CellService extends GlobalService {
     super();
   }
 
-  getCells(): Observable<any> {
+  getCells(isOrdered: boolean): Observable<any> {
```

Comment:
1. Can you change `isOrdered` by a more specific name ? (ex: `orderedByName`)
2. And i think it can be a good idea to change the boolean to a string so we can choose what type of `ordering` we want if one day we have more option. It will be me generic for all services.
3. A default value can be appreciate to not force developer to specify this attribut all the time if they don't want to.

Reply:
Good point !